generate_EthOn_doc.py

```diff
@@ -2,7 +2,7 @@
 # Command line tool for using alethio-scraper
 
 import argparse
-import ontospy
+from ontospy import core as ontospy
```

Comment:
Right, for some reason when I tried running the script I was getting an error flag because it couldn't locate the ontospy modules, which this hack seemed to resolve. Since we might not be using this script for long, I can change this back to `import ontospy`